samples/oauth_ccg_postman_pre_request_script/README.md

```diff
@@ -0,0 +1,30 @@
+# Postman CCG Pre-request Script Documentation
+
+## Description:
+
+The Postman CCG Pre-request Script simplifies the interaction process for developers working with Lighthouse CCG APIs. This script generates a signed JSON Web Token (JWT) and retrieves an access token, facilitating seamless communication with the Lighthouse API. The Postman collection provided uses the Benefits Claims API, but this does work with other CCG Lighthouse API's.
+
+## Steps to Use:
+
+1. Import the Postman Collection:
+    a. Download and import the "CCG Pre-request Script.postman_collection.json" into your Postman workspace.
+
+2. Set Global Variables:
+    a. In Postman, set the following global variables along with their corresponding values. Refer to the instructions on https://developer.va.gov/explore/api/benefits-claims/client-credentials for obtaining the values.
+
+    - aud
+    - clientId
+    - privatePem
+    - scope
```

Comment:
Recommend adding an example for each of these values here.  Especially scope so that the correct spacing is used for multiples.